express-notes-api/index.js

```diff
@@ -0,0 +1,127 @@
+const express = require('express');
+
+const app = express();
+
+const data = require('./data.json');
+
+const fs = require('fs');
+
+app.get('/api/notes', (req, res) => {
+  const arrayObjects = [];
+  const notes = data.notes;
+  for (const key in notes) {
+    arrayObjects.push(notes[key]);
+  }
+  res.send(arrayObjects);
+});
+
+app.listen(3000, () => {
+  // eslint-disable-next-line no-console
+  console.log('Express server listening on port 3000');
+});
+
+app.get('/api/notes/:id', (req, res) => {
+  const notes = data.notes;
+  const number = Number(req.params.id);
+  if (number < 0) {
+    res.status(400).send({
+      error: 'id must be a postiive integer'
+    });
+  }
```

Comment:
What if the number is `0`? Or a non-whole number? Or not a number to begin with?